docs/products/cassandra/howto/enable-cross-cluster-replication.rst

```diff
@@ -0,0 +1,174 @@
+
+.. code-block:: bash
+
+   avn service update service_name                     \
+      -c service_to_join_with=existing_service_name
+
+Enable CCR with API
+-------------------
+
+You can enable CCR for your new or existing Aiven for Apache Cassandra service using Aiven APIs.
+
+.. note::
+   
+   In this instruction, the `curl` command line tool is used to interact with Aiven APIs.
+
+.. tip::
+
+   Check out how to get started with Aiven APIs in :doc:`Aiven API </docs/tools/api>`.
+
+On a new service
+''''''''''''''''
+
+Use the `Create a service <https://api.aiven.io/doc/#tag/Service/operation/ServiceCreate>`_ API to create a new service with CCR enabled. When constructing the API request, add the ``user_config`` object to the request body and nest the ``service_to_join_with`` field inside it. As a value of the ``service_to_join_with`` field, set up a name of the service to which you'd like to connect to enable CCR and create a CCR service pair.
+
+.. code-block:: bash
+
+   curl --request POST                                                   \
+      --url https://api.aiven.io/v1/project/YOUR_PROJECT_NAME/service    \
+      --header 'Authorization: Bearer YOUR_BEARER_TOKEN'                 \
+      --header 'content-type: application/json'                          \
+      --data
+         '{
+            "cloud": "string",
+            "plan": "string",
+            "service_name": "new_service_name",
+            "service_type": "cassandra",
+            "user_config": {
+               "cassandra": {
+                  "datacenter": "datacenter_name"
+               },
+               "service_to_join_with": "service_name"
+            }
+         }'
+
+On an existing service
+''''''''''''''''''''''
+
+Use the `Update service configuration <https://api.aiven.io/doc/#tag/Service/operation/ServiceUpdate>`_ API to modify the configuration of your existing service so that it has CCR enabled. When constructing the API request, add the ``user_config`` object to the request body and nest the ``service_to_join_with`` field inside it. As a value of the ``service_to_join_with`` field, set up a name of the service to which you'd like to connect to enable CCR and create a CCR service pair.
+
+.. note::
+
+   To enable CCR on your service, you can connect to services residing only within datacenters that are different from the datacenter used by your service.
+
+.. code-block:: bash
+
+   curl --request PUT                                                                     \
+      --url https://api.aiven.io/v1/project/YOUR_PROJECT_NAME/service/YOUR_SERVICE_NAME   \
+      --header 'Authorization: Bearer YOUR_BEARER_TOKEN'                                  \
+      --header 'content-type: application/json'                                           \
+      --data
+         '{
+            "user_config": {
```

Comment:
datacenter is necessary here as well, otherwise you risk getting the 400 because it's `aiven` by default